spec/requests/set_user_after_login_spec.rb

```diff
@@ -17,6 +17,14 @@
     expect(new_user.last_name).to eq("Jetsonian")
   end
 
+  it "sends welcome email to a new user" do
```

Comment:
should we perhaps wrong a spec that ensures we don't send a welcome email to a user on login when the user is not new?